spec/helpers/holds_helper_spec.rb

```diff
@@ -96,4 +98,21 @@
       end
     end
   end
+
+  describe '#default_pickup_by_date' do
+    before do
+      # Instantiate two instances of DateTime before it is doubled.
+      arbitrary_datetime
+      arbitrary_datetime_plus
+      date_time = class_double('DateTime')
```

Comment:
👀 🆒

Reply:
if you don't instantiate the two arbitrary date times then the class double prevents them from being created, so order is important